EventsExpress/ClientApp/src/actions/event/event-add-action.js

```diff
@@ -45,7 +46,8 @@ export function publish_event(eventId) {
         let response = await api_serv.publishEvent(eventId);
         if (response.ok) {
             dispatch(setEventSuccess(true));
-            dispatch(getEvent(eventId));
+            dispatch(get_event(eventId));
```

Comment:
Yes, we are. Because the get_event method accepts an event id as a parameter, which comes to the publish_event method, while the getEvent takes the whole object. That's why we need to use exactly get_event method. Everything works as expected with these changes.